src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -209,4 +213,38 @@ private void closeMoreInfoAboutEqualShareModal() {
         waitForElementToDisappear(By.id("more-info-equal-share-modal"));
     }
 
+    public void addFeedbackResponseComment(String addResponseCommentId, String commentText) {
+        WebElement addResponseCommentForm = browser.driver.findElement(By.id(addResponseCommentId));
+        WebElement parentContainer = addResponseCommentForm.findElement(By.xpath("../.."));
+        WebElement showResponseCommentAddFormButton = parentContainer.findElement(By.id("button_add_comment"));
+        click(showResponseCommentAddFormButton);
+        WebElement editorElement = waitForElementPresence(By.cssSelector("#" + addResponseCommentId + " .mce-content-body"));
+        waitForRichTextEditorToLoad(editorElement.getAttribute("id"));
+        fillRichTextEditor(editorElement.getAttribute("id"), commentText);
+    }
+
+    public void editFeedbackResponseComment(String commentIdSuffix, String newCommentText) {
+        WebElement commentRow = browser.driver.findElement(By.id("responseCommentRow" + commentIdSuffix));
+        click(commentRow.findElements(By.tagName("a")).get(1));
+        fillRichTextEditor("responsecommenttext" + commentIdSuffix, newCommentText);
+    }
+
+    public void deleteFeedbackResponseComment(String commentIdSuffix) {
+        WebElement commentDeleteButton = browser.driver.findElement(By.id("commentdelete" + commentIdSuffix));
+        click(commentDeleteButton);
+        waitForConfirmationModalAndClickOk();
+        ThreadHelper.waitFor(1500);
```

Comment:
Should be fine then, we'll leave it to be fixed in another issue (I believe there's already one open for this, though it's pretty old)